packages/tailwind-config/index.ts

```diff
@@ -8,8 +8,8 @@ export default {
     './components/**/*.{ts,tsx}',
     './app/**/*.{ts,tsx}',
     './src/**/*.{ts,tsx}',
-    '../../packages/ui/components/**/*.{ts,tsx}',
     './shared/**/*.{ts,tsx}',
+    './node_modules/@penumbra-zone/ui/**/*.js'
```

Comment:
This line makes UI package work correctly with Tailwind.

Why: Tailwind performs JIT compilation to compose and export to a CSS file all used classes. In case of UI package, we have many dynamic classes, which are not parsed without this line change. For example, in `<Button variant='gradient'>` the variant dynamically adds classes and, to process them by JIT, users of UI package need to point Tailwind config to the node_modules.

Of course, it requires users to use Tailwind. If we want to change, we need to update the build process to export the JS files along with a CSS. This file must collect all styles from all components because otherwise users will have to import `button.css`, `table.css`, etc.